config/cache.yml

```diff
@@ -0,0 +1,16 @@
+default: &default
+  store_options:
+    # Cap age of oldest cache entry to fulfill retention policies
+    max_age: <%= 14.days.to_i %>
```

Comment:
This `max_age` is the default if you don't set it. I don't think it matters at this point because the only caching we currently do for historical trends is updated every 24 hours.

But we can make it 30 or 60 days if preferred.